### DISCUSSION TITLE
You don't have permission to access the current chat

### DISCUSSION THREAD

**Discussion by Creativologist** [LiveHelperChat/livehelperchat#2039] [Q&A] [answered]
This must be a newer feature as we haven't upgraded for a while.  How do I fix that?

**Comment by Creativologist** [comment]
Any idea on how to fix this?

"You do not have permission to access the current chat or chat was merged/deleted!"

(wasn't merged or deleted)

**Reply by remdex** [reply, marked as answer]
Please read. There is new permissions in new versions
https://doc.livehelperchat.com/docs/chat/#permissions-influencing-chat-opening-permissions

Also see permissions for listing chats
https://doc.livehelperchat.com/docs/chat/list#introduction

**Reply by Creativologist** [reply]
Thanks.  Updating my settings.ini file corrected both problems.
